modules/api-gateway/main.tf

```diff
@@ -21,39 +21,6 @@ resource "aws_api_gateway_rest_api" "orca_api" {
   }
 }
 
-resource "aws_api_gateway_rest_api_policy" "orca_api_resource_policy" {
-  rest_api_id = aws_api_gateway_rest_api.orca_api.id
-
-  policy = <<EOF
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Principal": {
-        "AWS": "*"
-      },
-      "Action": "execute-api:Invoke",
-      "Resource": "${aws_api_gateway_rest_api.orca_api.execution_arn}"
-    },
-    {
-      "Effect": "Deny",
-      "Principal": {
-        "AWS": "*"
-      },
-      "Action": "execute-api:Invoke",
-      "Resource": "${aws_api_gateway_rest_api.orca_api.execution_arn}",
-      "Condition": {
-        "StringNotEquals": {
-          "aws:SourceVpc": "${var.vpc_id}"
-        }
-      }
-    }
-  ]
-}
-EOF
-}
-
```

Comment:
these changes were not added in orca-437. Do we still want to remove it?
This policy denies API invoke calls made outside vpc.

Reply:
It would, if NGAP didn't remove it. Best I can tell, this was never left active for more than a few milliseconds.

Reply:
I checked my api resource policy and that looks way different than what I added. So NGAP changed the policy in the backend without informing us.

Reply:
Looks like communication between teams is becoming a problem and this is not the first time we are facing the consequences :)

Reply:
FYi @Quarriers89 @rykearney